backend/github.js

```diff
@@ -96,3 +96,35 @@ exports.deleteHook = function (name, hook, accessToken, callback) {
     }
   });
 };
+
+/**
+ * Create status to the commit
+ * @param sha: commit sha
+ * @param repositoryName: name of the repository. example: fossasia/yaydoc
+ * @param state: state of the commit. accepted parameters are pending, success, failure, error
+ * @param description: short description of the status
+ * @param personalAccessToken: access token of the user who having access to the repository
+ * @param callback
+ */
+exports.createStatus = function(sha, repositoryName, state, description, personalAccessToken, callback) {
+  request.post({
+    url: `https://api.github.com/repos/${repositoryName}/statuses/${sha}`,
+    headers: {
+      'User-Agent': 'Yaydoc',
+      'Authorization': 'token ' + personalAccessToken
+    },
+    "content-type": "application/json",
+    body: JSON.stringify({
+      state: state,
+      target_url: `https://yaydoc.herokuapp.com`,
+      description: description,
+      context: "Yaydoc CI"
+    })
+  }, function(error, response, body) {
+    if (error!== null) {
+      callback({description: 'Unable to create status'}, null)
```

Comment:
`return callback().....` 
Making this change, remove the following else, resulting in 

```
if (error) {
  return callback({description: 'Unable to create status'}, null);
}
callback(null, JSON.parse(body));
```